spec/api/v1/external_users/representation_order_spec.rb

```diff
@@ -35,15 +35,17 @@
   end
 
   describe "POST #{endpoint(:representation_orders)}" do
-    def post_to_create_endpoint(submission_date = Date.new(2017, 7, 1))
-      Timecop.freeze(submission_date) { post endpoint(:representation_orders), valid_params, format: :json }
+    def post_to_create_endpoint
+      post endpoint(:representation_orders), valid_params, format: :json
     end
 
+    subject { post endpoint(:representation_orders), valid_params, format: :json }
+
     include_examples "should NOT be able to amend a non-draft claim"
 
     context 'when representation_order params are valid' do
       it "should create fee, return 201 and expense JSON output including UUID" do
-        post_to_create_endpoint
+        subject
```

Comment:
would prefer named `subject` at line 42 - for clarity and to avoid potential clashes. is also a rubocop preference (we are excluding specs from rubocop entirely at the moment i think due to the number of violations i think)

Reply:
Sorry, I put `subject` in at one point and then thought I had taken it out again later, when I understood better what was going on. I'll change it back.